### DISCUSSION TITLE
Text partially skewed in document

### DISCUSSION THREAD

**Discussion by KFoster2** [ImageMagick/ImageMagick#2051] [Help]
Hello,

I have a document where the text is skewed in certain areas but straight in the rest. Sample attached.

Is there any way to straight the skewed parts and leave the straight ones as they are?

I know this is challenging but hope there is a solution for that.
![sample](https://user-images.githubusercontent.com/62470748/82578058-2cf1eb00-9b8c-11ea-9fab-6c260b97ac0f.jpg)

**Comment by fmw42** [comment]
I know of no automatic way to do that. Crop the skewed part out, deskew it (-deskew), then insert it back in.
